docs/perf/device_tuning.rst

```diff
@@ -30,26 +30,51 @@ a PR and include a screenshot from Wandb or other verifiable evidence.
 7B
 ~~~
 
-.. table::
-   :widths: auto
-
-   ====== ====== ====== ======== ====== ====== ======
-   tag    model  task   resource train  infer  link
-   ====== ====== ====== ======== ====== ====== ======
-   \      \      \        \      \      \
-   ====== ====== ====== ======== ====== ====== ======
-
-13B
+.. list-table::
+    :widths: auto
+    :header-rows: 1
+    
+    * - Tag
+      - Model
+      - Task
+      - Resource
+      - Train
+      - Infer
+      - Link
+      - Contributor
+    * - MIN
+      - Qwen2-7B
+      - GRPO
+      - 2*H800
+      - fsdp
+      - vllm0.8.2
+      - `qwen2-7b_grpo_2_h800_fsdp_vllm <../../examples/tuning/7b/qwen2-7b_grpo_2_h800_fsdp_vllm.sh>`_
```

Comment:
maybe we should include dataset or seqlen in the name since seqlen affects memory requirement